--- ./packages/vm/lib/state/stateManager.ts
+++ ./packages/vm/lib/state/stateManager.ts
@@ -5,7 +5,7 @@ import * as utils from 'ethereumjs-util'
 import BN = require('bn.js')
 import { encode, decode } from 'rlp'
 import Common from 'ethereumjs-common'
-import { genesisStateByName } from 'ethereumjs-common/dist/genesisStates'
+import { genesisStateByName } from 'ethereumjs-common/genesisStates'
 import Account from 'ethereumjs-account'
 import Cache from './cache'
 import { ripemdPrecompileAddress } from '../evm/precompiles'
